traffic_ops/traffic_ops_golang/servicecategory/servicecategories.go

```diff
@@ -187,3 +192,313 @@ WHERE name=$2 RETURNING last_updated`
 func deleteQuery() string {
 	return `DELETE FROM service_category WHERE name=:name`
 }
+
+// Get [Version : V5] function Process the *http.Request and writes the response. It uses GetServiceCategory function.
+func Get(w http.ResponseWriter, r *http.Request) {
+	inf, userErr, sysErr, errCode := api.NewInfo(r, nil, nil)
+	if userErr != nil || sysErr != nil {
+		api.HandleErr(w, r, inf.Tx.Tx, errCode, userErr, sysErr)
+		return
+	}
+	defer inf.Close()
+
+	code := http.StatusOK
+	useIMS := false
+	config, e := api.GetConfig(r.Context())
+	if e == nil && config != nil {
+		useIMS = config.UseIMS
+	} else {
+		log.Warnf("Couldn't get config %v", e)
+	}
+
+	var maxTime time.Time
+	var err error
+	var scList []tc.ServiceCategoryV5
+
+	tx := inf.Tx
+
+	scList, maxTime, code, err = GetServiceCategory(tx, inf.Params, useIMS, r.Header)
+	if code == http.StatusNotModified {
+		w.WriteHeader(code)
+		api.WriteResp(w, r, []tc.ServiceCategoryV5{})
+		return
+	}
+
+	if code == http.StatusBadRequest {
+		api.HandleErr(w, r, inf.Tx.Tx, http.StatusBadRequest, err, nil)
+		return
+	}
+
+	if err != nil {
+		api.HandleErr(w, r, inf.Tx.Tx, http.StatusInternalServerError, nil, errors.New(err.Error()))
```

Comment:
we shouldn't be doing `errors.New(err.Error())`, just simply pass the error.

Reply:
Changed it to only err